src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_PRICE_PER_DAY = 40;
+  const BASIC_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  const basePrice = days * BASE_PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
```

Comment:
The logic here applies only the long-term discount if the number of days is 7 or more. However, the task description suggests that both the long-term and basic discounts should be applied if the conditions are met. Consider revising this logic to ensure both discounts are applied when applicable.